### PR TITLE
Security updates

### PR DESCRIPTION
**Pull request by skeltonm**
Removed full file path from ForgeBox installer and media manager location bar
Prevents server paths from being exposed in the ContentBox admin

### CODE REVIEW

**Comment by bdw429s** [comment]
Thanks for the pull @skeltonm.  Just to clarify, does this pull close any specific vulnerabilities or exploits in ContentBox, or is this just improvement to help make ContentBox more secure?

**Comment by skeltonm** [comment]
Yes and no.  This doesn't fix any security exploits for attackers trying to break into the admin.  This is targeting at hiding the server path from users who are using ContentBox admin.  In our case, we are trying to use Amazon S3 for storing our media files.  We setup the S3 path as a Lucee mapping so as to hide the S3 keys from the users who use the admin.  Previous the the media manager update I made, it would show the entire S3 path including the S3 keys.  For the ForgeBox update, we felt like showing the full server path is unnecessary and better to simply keep that information hidden.

Does that make sense?

**Comment by bdw429s** [comment]
Thanks-- that makes sense.  You're essentially limiting the information about the file system that's disclosed to the ContentBox user.  If they're using a hosted ContentBox install, then the idea is that it's none of their business where the files are ultimately going.   cc/ @lmajano

File: modules/contentbox/models/util/ForgeBoxInstaller.cfc [hunk 1]
```diff
@@ -72,15 +72,16 @@ component singleton{
 				overwriteFiles	= "true" 
 			);
 
-			log.append( "Archive uncompressed and installed at #destination#. Performing cleanup.<br />" );
+			var cleanDestination = replaceNoCase( destination, REReplace( expandPath( "/" ), "\\", "/", "all" ), "" );
```
Comment (lmajano) [inline modules/contentbox/models/util/ForgeBoxInstaller.cfc]: Can you just remove the `#destination#`, let's clean that up

Comment (skeltonm) [inline modules/contentbox/models/util/ForgeBoxInstaller.cfc]: Sure thing.  I updated that file.

**Pull request merged**